docs/fides/docs/language/taxonomy/data_categories.md

```diff
@@ -6,18 +6,17 @@ Data Categories are hierarchical labels used to describe the type of data proces
 
 ## Object Structure
 
-**fides_key**<span class="required"/>_string_
+**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_constrained string_
 
 A string token that uniquely identifies this Data Category. The value is a dot-separated concatenation of the `fides_key` values of the resource's ancestors plus a final element for this resource:
 
 `grandparent.parent.this_data_category`
 
-The final element (`this_data_category`) may only contain alphanumeric characters and underbars (`[A-Za-z0-9_]`). The dot character is reserved as a separator.
-
+The final element (`this_data_category`) may only contain alphanumeric characters and underbars (`[A-Za-z0-9_.-]`). The dot character is reserved as a separator.
```

Comment:
```suggestion
The final element (`this_data_category`) may only contain alphanumeric characters, `_`, `.`, and `-` (it will be evaluated against the regular expression: `[A-z0-9_.-]`). The dot character is reserved as a separator.
```